src/components/_proxied-dot-io/common/dev-dot-opt-in/index.tsx

```diff
@@ -48,19 +59,42 @@ export default function DevDotOptIn() {
 	}
 
 	return (
-		<div className={s.container}>
-			<IconAlertCircleFill16 className={s.icon} />
-			<p className={s.alert}>
-				The {name} website is being redesigned to help you find what you are
-				looking for more effectively.
-				<a
-					className={s.optInLink}
-					href={getDevDotLink(slug, asPath)}
-					onClick={handleOptIn}
-				>
-					Join the Beta
-				</a>
-			</p>
+		<div className={s.root}>
+			<div className={s.alertContainer}>
+				<div className={s.icon}>
+					<IconAlertCircle16 />
+				</div>
+				<div className={s.contentContainer}>
+					<p className={s.title}>
```

Comment:
💭 You can use the `Text` component to get a lot of the type styles for free that you had to write.

Reply:
Done!